apierr/errors.go

```diff
@@ -104,6 +104,11 @@ func (apiError *APIError) IsRetriable(ctx context.Context) bool {
 	if apiError.IsTooManyRequests() {
 		return true
 	}
+	// Retry when the API is unavailable. This includes "No webapps are available
+	// to handle your request. Please try again later." from the API gateway.
+	if apiError.StatusCode == http.StatusServiceUnavailable {
+		return true
+	}
```

Comment:
This could be too broad. Could we make it explicit through the transient error messages?

Reply:
Generally retrying on 5xx is the right thing to do

Reply:
Not for all of them. E.g. 504 Gateway Timeout for a POST must not automatically retry. If the unavailability means the request may or may not have been seen by a backend it's tricky. And we don't differentiate by HTTP method here.

Reply:
I don't think retrying on 504 (Gateway Timeout) is safe, as that request could have been handled by the backend service. However, 503 (Service Unavailable) should be safe, since that implies that the request never was processed by a backend service.

Reply:
Good point. Mozilla agrees: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Retry-After.

Reply:
Yup, that was my reference too :)